rever/activities/bibtex.xsh

```diff
@@ -0,0 +1,75 @@
+"""Activity for bumping version."""
+import datetime
+import os
+
+from rever import vcsutils
+from rever.activity import Activity
+try:
+    import bibtexparser
+except ImportError:
+    pass
+
+# $ACTIVITIES = ['bibtex']
+
+def render_authors(authors):
+    """Parse a list of of tuples of authors into valid bibtex
+
+    Parameters
+    ----------
+    authors: list of str
+        The authors eg ['Your name in nicely formatted bibtex'].
+        Please see ``<http://nwalsh.com/tex/texhelp/bibtx-23.html>`` for
+        information about how to format your name for bibtex
+
+    Returns
+    -------
+    str:
+        Valid bibtex authors
+    """
+    if len(authors) == 1:
+        return ' '.join(authors[0])
```

Comment:
Why do you want to add a space between every character of an authors name?  Shouldn't this just return `authors[0]`